use crypto_bros::error::Result;
pub fn main() -> Result<()> {
    solve()
}

fn solve() -> Result<()> {
    Ok(())
}
